slices/apparmor.yaml

```diff
@@ -0,0 +1,199 @@
+package: apparmor
+
+essential:
+  - apparmor_copyright
+
+slices:
+  # Also relies on debconf, however it is solely used by install hook
+  # to determine if any non default homedir has been configured.
+  bins:
+    essential:
+      - apparmor_config
+      - apparmor_profiles
+      - libc6_libs
+      - lsb-base_bins
+    contents:
+      /etc/init.d/apparmor:
+      /sbin/apparmor_parser:
+      /usr/bin/aa-enabled:
+      /usr/bin/aa-exec:
+      /usr/bin/aa-features-abi:
+      /usr/sbin/aa-load:
+      /usr/sbin/aa-remove-unknown:
+      /usr/sbin/aa-status:
+      /usr/sbin/aa-teardown:
+      /usr/sbin/apparmor_status:
+
+  config:
+    contents:
+      /etc/apparmor/parser.conf:
+      /usr/lib/sysctl.d/10-apparmor.conf:
+
+  extras:
+    contents:
+      /lib/apparmor/apparmor.systemd:
+      /lib/apparmor/profile-load:
+      /lib/apparmor/rc.apparmor.functions:
+      /usr/lib/systemd/system/apparmor.service:
+
+  profiles:
+    contents:
+      /etc/apparmor.d/1password:
+      /etc/apparmor.d/Discord:
+      /etc/apparmor.d/MongoDB_Compass:
+      /etc/apparmor.d/QtWebEngineProcess:
+      /etc/apparmor.d/abi/3.0:
+      /etc/apparmor.d/abi/4.0:
+      /etc/apparmor.d/abi/kernel-5.4-outoftree-network:
+      /etc/apparmor.d/abi/kernel-5.4-vanilla:
+      /etc/apparmor.d/abstractions/**:
+      /etc/apparmor.d/brave:
+      /etc/apparmor.d/buildah:
+      /etc/apparmor.d/busybox:
+      /etc/apparmor.d/cam:
+      /etc/apparmor.d/ch-checkns:
+      /etc/apparmor.d/ch-run:
+      /etc/apparmor.d/chrome:
+      /etc/apparmor.d/code:
+      /etc/apparmor.d/crun:
+      /etc/apparmor.d/devhelp:
+      /etc/apparmor.d/element-desktop:
+      /etc/apparmor.d/epiphany:
+      /etc/apparmor.d/evolution:
+      /etc/apparmor.d/firefox:
+      /etc/apparmor.d/flatpak:
+      /etc/apparmor.d/geary:
+      /etc/apparmor.d/github-desktop:
+      /etc/apparmor.d/goldendict:
+      /etc/apparmor.d/ipa_verify:
+      /etc/apparmor.d/kchmviewer:
+      /etc/apparmor.d/keybase:
+      /etc/apparmor.d/lc-compliance:
+      /etc/apparmor.d/libcamerify:
+      /etc/apparmor.d/linux-sandbox:
+      /etc/apparmor.d/local/README:
```

Comment:
are you missing the `/etc/apparmor.d/local/nvidia_modprobe`?

Reply:
and `/etc/apparmor.d/local/lsb_release`

Reply:
Nicely spotted!! I had forgotten those two